middleware/third_party/arduino/hardware/arduino/mt7697/libraries/MCS/src/MCS.cpp

```diff
@@ -483,4 +483,92 @@ bool MCSDisplayOnOff::value(void)
 void MCSDisplayOnOff::_dispatch(const String& params)
 {
     // do nothing for display channel
+}
+
+
+/* ----------------------------------------------------------------------------
+class MCSControllerInteger
+---------------------------------------------------------------------------- */
+
+MCSControllerInteger::MCSControllerInteger(const String& channel_id):
+MCSDataChannel(channel_id),
+mValue(0) // TODO: Intial value ?? N/A ??
```

Comment:
Still 0 is generally a good idea for default values. A common C idiom is that numerical members are always initialized to 0 as default, and is supported by [some language syntax](http://en.cppreference.com/w/cpp/language/zero_initialization).